src/main/java/com/amadeus/resources/HotelBooking.java

```diff
@@ -0,0 +1,18 @@
+package com.amadeus.resources;
+
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * An HotelBooking object as returned by the Hotel Booking API.
+ * @see com.amadeus.booking.HotelBookings#get()
+ */
+@ToString
+public class HotelBooking extends Resource {
+  protected HotelBooking() {}
+
+  private @Getter String type;
```

Comment:
I think the model misses some params (associatedRecord) (that maybe are not used yet but I can see them in the model in the swagger:
![image](https://user-images.githubusercontent.com/713173/76956106-add60180-6913-11ea-9cfa-e35bdea9f3c6.png)